arches/app/search/components/standard_search_view.py

```diff
@@ -0,0 +1,240 @@
+        search_query_object["query"].include("points")
+        search_query_object["query"].include("geometries")
+        search_query_object["query"].include("displayname")
+        search_query_object["query"].include("displaydescription")
+        search_query_object["query"].include("map_popup")
+        search_query_object["query"].include("provisional_resource")
+        search_query_object["query"].include("permissions")
+        load_tiles = get_str_kwarg_as_bool("tiles", self.request.GET)
+        if load_tiles:
+            search_query_object["query"].include("tiles")
+
+    def execute_query(self, search_query_object, response_object, **kwargs):
+        for_export = get_str_kwarg_as_bool("export", self.request.GET)
+        pages = self.request.GET.get("pages", None)
+        total = int(self.request.GET.get("total", "0"))
+        resourceinstanceid = self.request.GET.get("id", None)
+        dsl = search_query_object["query"]
+        if for_export or pages:
+            results = dsl.search(index=RESOURCES_INDEX, scroll="1m")
+            scroll_id = results["_scroll_id"]
+            if not pages:
+                if total <= settings.SEARCH_EXPORT_LIMIT:
+                    pages = (total // settings.SEARCH_RESULT_LIMIT) + 1
+                else:
+                    pages = SEARCH_RESULT_PAGES
+            for page in range(int(pages)):
+                results_scrolled = dsl.se.es.scroll(scroll_id=scroll_id, scroll="1m")
+                results["hits"]["hits"] += results_scrolled["hits"]["hits"]
+        else:
+            results = dsl.search(index=RESOURCES_INDEX, id=resourceinstanceid)
+
+        if results is not None:
+            if "hits" not in results:
+                if "docs" in results:
+                    results = {"hits": {"hits": results["docs"]}}
+                else:
+                    results = {"hits": {"hits": [results]}}
+        response_object["results"] = results
+
+    def post_search_hook(self, search_query_object, response_object, **kwargs):
+
+        dsl = search_query_object["query"]
+        response_object["reviewer"] = user_is_resource_reviewer(self.request.user)
+        response_object["timestamp"] = datetime.now()
+        response_object["total_results"] = dsl.count(index=RESOURCES_INDEX)
+        response_object["userid"] = self.request.user.id
+
+    def get_searchview_components(self):
+        search_components = [
+            available_component
+            for available_component in self._available_search_components
+            if available_component.componentname != "search-export"
+        ]
+        if user_is_resource_exporter(self.request.user):
+            search_components.extend(
+                [
+                    available_component
+                    for available_component in self._available_search_components
+                    if available_component.componentname == "search-export"
+                ]
+            )
+
```

Comment:
you should use the propertie (available_search_components) of the class instead of the private variable (_available_search_components)
```suggestion
        search_components = [
            available_component
            for available_component in self.available_search_components
            if available_component.componentname != "search-export"
        ]
        if user_is_resource_exporter(self.request.user):
            search_components.extend(
                [
                    available_component
                    for available_component in self.available_search_components
                    if available_component.componentname == "search-export"
                ]
            )

```